pom.xml

```diff
@@ -3,17 +3,93 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
     <groupId>TradeBotAPI</groupId>
     <artifactId>TradeBot</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <java.version>17</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <spring-boot.version>3.0.0</spring-boot.version>
+        <spring.cloud-version>2022.0.0</spring.cloud-version>
+        <jackson.version>4.0.0</jackson.version>
+    </properties>
+
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-autoconfigure</artifactId>
-            <version>2.7.4</version>
+            <version>${spring-boot.version}</version>
+        </dependency>
```

Comment:
please, use parent

org.springframework.boot
spring-boot-starter-parent
3.0.0
and remove version from dependencies

Reply:
Done